test/integration/dra/dra_test.go

```diff
@@ -115,6 +118,11 @@ func TestDRA(t *testing.T) {
 	// change the global DefaultFeatureGate. For each configuration,
 	// multiple tests can run in parallel as long as they are careful
 	// about what they create.
+	//
+	// Each configuration starts with two Nodes (ready, sufficient RAM and CPU for multiple pods)
+	// and no ResourceSlices. To test scheduling, a sub-test must create ResourceSlices.
```

Comment:
... and call `startScheduler`. That one is actually a problem because we need the scheduler to be a singleton and the tests to run in parallel.

I'll probably add ref-counting to start the scheduler on-demand and tear it down after the last user is gone.